aten/src/ATen/cuda/llvm_complex.cpp

```diff
@@ -477,6 +477,14 @@ operator!=(const _Tp& __x, const complex<_Tp>& __y)
     return !(__x == __y);
 }
 
+template<class _Tp>
+inline constexpr
+bool
+operator&&(const _Tp& __x, const complex<_Tp>& __y)
```

Comment:
I think you will have to add an overload for  `bool operator&&(complex<T>&, complex<T>&)`

Reply:
Ref : https://github.com/pytorch/pytorch/runs/5789888945?check_suite_focus=true#step:9:20384

Reply:
Hi, @kshitij12345!
Thanks for your input! :)

I've added it. Looking into the other operators. I considered the situation in three different ways: both the inputs are complex, or if either args are complex.

Is this the right way? Thanks!